src/services/walletConnectService.ts

```diff
@@ -524,6 +526,61 @@ export class WalletConnectService {
     return result;
   };
 
+  /**
+   *
+   * @param action Wallet Action that the wallet should take (e.g. `switchToGroup`)
+   * @param payload JSON payload for wallet to handle
+   * @param description (optional) Additional information for wallet to display
+   * @param method (optional) What method is used to send this message
+   */
+  sendWalletAction = async ({
```

Comment:
Thoughts on keeping this method private and instead adding the specific "actions" as service methods for dApps to use?